packages/uswds-core/src/js/index.js

```diff
@@ -5,6 +5,7 @@ const comboBox = require("../../../usa-combo-box/src/index");
 const datePicker = require("../../../usa-date-picker/src/index");
 const dateRangePicker = require("../../../usa-date-range-picker/src/index");
 const fileInput = require("../../../usa-file-input/src/index");
+const inputMask = require("../../../usa-input-mask/src/index");
```

Comment:
Can we put these in alphabetical order?

Reply:
Alphabetized... Turns out a couple more needed it (navigation was after search) P.S. Love the Alphabetical Sorter VSCode extension!